contentctl/objects/constants.py

```diff
@@ -136,4 +136,11 @@
 RBA_OBSERVABLE_ROLE_MAPPING = {
     "Attacker": 0,
     "Victim": 1
-}
\ No newline at end of file
+}
+
+# TODO (cmcginley): do these belong here? or in inspect.py?
+ESCU_APP_UID = 3449
+
+DOWNLOADS_DIRECTORY = "downloads"
```

Comment:
Do we want to add this directory to the `.gitignore` in the GitLab and GitHub repos?

Reply:
1. ok cool; i'll make a TODO item in this PR indicating a follow up should be made to both the public and private repos which adds this directory to the .gitignore
2. that's a great question. I definitely think that downloads for app installs and this sort of temporary supplemental data download should live in different places. I think I like the idea of downloads being the parent directory, w/ sub dirs as needed for different use-cases. So `downloads/apps/` and `downloads/inspect/' maybe to keep things concise?

Reply:
That said, I think making that change to apps downloads feels out of scope and something to be addressed in a different PR (we could create an issue to track). Happy to have `inspect` start downloading to a subdir in this PR though, if you agree.

Reply:
I have marked this as a future enhancement for all of this work.
I don't think we need to make any of these changes right now.
https://github.com/splunk/contentctl/issues/284